osf/models/brand_assets.py

```diff
@@ -0,0 +1,13 @@
+from osf.models.base import BaseModel
+from django.db import models
+from colorfield.fields import ColorField
+
+
+class BrandAssets(BaseModel):
+    """
+    This model holds data custom styled assets for providers
+    """
+
+    hero_logo = models.URLField()
+    topnav_logo = models.URLField()
+    hero_background = ColorField()
```

Comment:
`hero_background` should be another `URLField` -- it's the background image.

it's an understandable mistake -- how about renaming these fields to make it clear? `hero_logo_image`, `navbar_logo_image`, and `hero_background_image`